src/stories/pages/AddExperiment/AddExperiment.js

```diff
@@ -0,0 +1,137 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+import ExperimentServices
+  from '../../../firebase/CRUDServices/experimentServices';
+
+import { FormItem } from '../../components/FormItem/FormItem';
+import { Form } from '../../components/Form/Form';
+import { Button } from '../../components/Button/Button';
+import { Modal } from '../../components/Modal/Modal';
+
+/**
+ * Component for AddExperiment page.
+ * @param {string} theme theme of the parent component.
+ * @param {string} buttonText text to appear on the open button.
+ * @component
+ * @return {object} (
+ *   <AddExperiment isDarkTheme={isDarkTheme}>
+ * )
+ */
+
+export const AddExperiment = ({ theme, buttonText }) => {
+  const [error, setError] = useState('');
+  const [title, setTitle] = useState('');
+  const [shortDesc, setShortDesc] = useState('');
+  const [longDesc, setLongDesc] = useState('');
+  const [qualtrics, setQualtrics] = useState('');
+  const [spych, setSpych] = useState('');
+
+  const postEx = (e) => {
+    e.preventDefault();
+
+    const service = ExperimentServices.postExperiment();
+
+    const experiment = {
+      title,
+      shortDesc,
+      longDesc,
+      qualtrics,
+      spych,
+    };
+
+    // To be removed as firebase ID should be automatically generated
+    const experimentId = Math.random();
+
+    service.postExperiment(experimentId, experiment).then((response) => {
```

Comment:
I guess if the ID is not needed by Firebase we could just remove it from here? But I guess the API would need to be changed too. If that's the case I think this is fine to leave it as it is